build/common.sh

```diff
@@ -711,6 +711,7 @@ function kube::build::copy_output() {
   kube::build::rsync \
     --prune-empty-dirs \
     --filter='- /_temp/' \
+    --filter='- /_output/dockerized/go/**' \
```

Comment:
I believe we don't need a GOPATH copy, but if we do perhaps it would be better to ignore `/_output/dockerized/go/cache/**` instead. 

A GOTOOLCHAIN=go1.22.1 for example is stored in `_output/dockerized/go/cache/mod/golang.org/toolchain@v0.0.1-go1.22.1.linux-amd64/`

Reply:
move this down next to `/_output/dockerized/bin/**` please

Reply:
ack